config/rbac/deployment_manager_role.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: deployment-manager-role
```

Comment:
This is used to give the provider controller permissions to manage the provider service deployment.